solution/text-extractor/backends/web.py

```diff
@@ -7,11 +9,14 @@
 class WebBackend(FileBackend):
     backend = "web"
 
-    def get_file(self, uri: str) -> bytes:
+    def get_file(self, temp_directory: str, uri: str) -> bytes:
```

Comment:
We store the response content in an object then delete it at the end so same as the way it is done in s3.